tutorials/_Quick_start.md

```diff
@@ -0,0 +1,78 @@
+*I recommend VScode but any terminal with git will be fine*. *If you do not have git set up and don't know how to do it see the sparse instructions at [RSE git](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Git-Setup) or ask chat gpt "how can I setup git in \[vscode | wsl | windows\] and connect my github account".*
+
+- Fork the [REMIND repo](https://github.com/remindmodel/remind/tree/develop) to your github account
+	- You can uncheck "develop only" but we will fix this below
+- clone the code (instead of a full github clone and checkout)
+	-  `cloneremind https://github.com/<yourusername>/remind.git DESTINATION_FOLDER`
+	- why? this will keep the data references from the HPC and reduce the size of the operation
+- Check the state of your repo
+	- navigate to your repo `cd myrepo`
+	- `git remote -v`
+- Switch to a stable release
+	- run `git fetch --all --tags` if you forgot to get the tags
+	- switch to a (hopefully) e.g. stable release `git checkout tags/v3.4.0 -b release-3.4.0`
+
+ See tutorial [01](https://github.com/remindmodel/remind/blob/develop/tutorials/01_GettingREMIND.md) for more details.
+ 
+# 2. Preliminaries
+
+## Overview of Remind code
+Read [REMIND - REgional Model of INvestments and Development – Overview](https://rse.pik-potsdam.de/doc/remind/3.2.0/) or equivalently the comment starting `main.gams` . This will be helpful in terms of understanding the code.
+This [tutorial](https://www.gams.com/latest/docs/UG_Tutorial.html) is recommended by the group.
+## !! Renv !!
+Note that REMIND **releases do not come with a locked environment** ([see the associated issue, which may be closed by the time you start](https://github.com/remindmodel/development_issues/issues/528)). This means that you will need to restore the version corresponding to your release: for example `make historic-reqs v=2024-12-11` where the date should be replaced with that of the release (see [here](https://github.com/remindmodel/development_issues/issues/528)) 
+
+# 3. Your first run
+*Note <some_text> indicate place holders*
+*To learn about the HPC and slurm have a look at [the IT HPC guide](https://www.pik-potsdam.de/en/institute/about/it-services/hpc/hpc-2024/hpc-2024-documentation/hpc-2024-documentation)* (you can use remind without knowing anything about high performance computers or the slurm job manager)
+
+- make a copy of `config/scenario_config.csv` -> `config/<myfirstrun_cfg>.csv` 
+	-  keep only 2-3 rows, say those in table1
+	-  **NB: the config uses German delimiters `;`**
+	- `scenario_config.csv` represents all the possible scenarios. The others don't really belong in a code repo but are there
+	- How the config.csv work is explained in [the REMIND tutorial 03](https://github.com/remindmodel/remind/blob/develop/tutorials/03_RunningBundleOfRuns.md)
+	- do not edit `default.cfg` which is actually an R script
+- check your run
+	- `Rscript start.R --test config/<myfirstrun_cfg>.csv`
+	- press enter to (mock) submit to the priority queue
```

Comment:
```suggestion

```
Please remove this line, I will change this, it makes no sense to ask the user at this point